src/app/zap-templates/zcl/data-model/chip/washer-controls-cluster.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="WasherControlsEnum" type="ENUM8">
+    <cluster code="0x0061"/>
+    <item name="Spin" value="0x00"/>
+    <item name="Rinse" value="0x01"/>
+    <item name="Error"   value="0x03"/>
+  </enum>
+
+  <enum name="ErrorStateEnum" type="ENUM8">
+    <cluster code="0x0061"/>
+    <item name="NoError"                    value="0x00"/>
+    <item name="CommandInvalidInState"      value="0x01"/>
+  </enum>
+
+  <struct name="SpinSpeedIndex">
+    <cluster code="0x0061"/>
+    <item name="WasherControlsID"    type="WasherControlsEnum" optional="false"/>
+    <item name="WasherControlsLabel" type="CHAR_STRING" optional="false"/>
+  </struct>
+
+  <struct name="ErrorStateStruct">
+    <cluster code="0x0061"/>
+    <item name="ErrorStateID" type="ErrorStateEnum" optional="false" />
+    <item name="ErrorStateLabel" type="CHAR_STRING" optional="false" length="64"/>
+    <item name="ErrorStateDetails" type="CHAR_STRING" optional="false" length="64"/>
+  </struct>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Washer Controls</name>
+    <code>0x0061</code>
```

Comment:
This cluster id does not match the spec.

Reply:
It's fixed. Please review. Thank you